examples/manifold/plot_compare_methods.py

```diff
@@ -164,7 +164,7 @@ def add_2d_scatter(ax, points, points_color, title=None):
 
 md_scaling = manifold.MDS(
     n_components=n_components,
-    max_iter=50,
+    n_iter=50,
```

Comment:
wait I'm confused, MDS `max_iter` and not `n_iter`

Reply:
Uhm Indeed. they should have the code in common with TSNE because it is raising in main and this is the only occurrence of the of `max_iter`. I have to check if we can temporarily pass `n_iter=max_iter`.